docker/travis_test/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN zypper in -y -C \
        automake \
        gcc-c++ \
        git \
+       git \
```

Comment:
This is now duplicated. I suppose you have to remove `git` from `docker_requires` which apparently "inherits" from `test_requires` without de-duplication.

Reply:
It doesn't inherit, but here is the definition of the docker dependencies:
https://github.com/os-autoinst/os-autoinst/blob/master/dependencies.yaml#L13

So yes, it should be removed from `docker_requires`.